guides/howtos/Multi tenancy with foreign keys.md

```diff
@@ -216,11 +216,11 @@ create table(:comments) do
   add :org_id, :integer, null: false
 
   add :post_id,
-      references(:posts, with: [org_id: :org_id]),
+      references(:posts, with: [org_id: :org_id], match: :full),
```

Comment:
This is not supported by all DBs, so we should revert it?

Reply:
Reverted